astropy/coordinates/earth.py

```diff
@@ -1,6 +1,5 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
+from __future__ import absolute_import, division, print_function
```

Comment:
FWIW you can also make cross-compatible literals by writing, for example `str('x'), str('y'), str('z')`--on Python 3 this will remain unicode strings.  On Python 2 (with `unicode_literals`) these will be converted to ascii strings.

Though looking over this module in whole I think it should be safe enough to drop `unicode_literals` entirely.